webrtc.html

```diff
@@ -8852,6 +8854,34 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+	  <li>If the selector is a MediaStreamTrack, return the following objects:
+	    <ul>
+	      <li>
+		The RTCMediaStreamStats corresponding to the selector.
```

Comment:
I think so. But as long as we're returning more than track stats it makes sense to return the RTCMediaStreamStats too (stream stats whose trackIds contains the RTCMediaStreamTrackStats.id of the track).